docs/proposals/20230612-node-os-image-caching.md

```diff
@@ -205,9 +242,19 @@ It should be tested primarily in isolation as other components shouldn’t affec
 
 ### Graduation Criteria [optional]
 
-alpha - The feature is initially constructed and toggleable with an environment variable
-beta - The feature has e2e tests implemented and is more integrated and seamless with the project.
-At this point we should also move the
+Alpha:
+
+The feature is initially constructed and toggleable with an environment variable
+
+Beta:
+
+The feature has e2e tests implemented and is more integrated and seamless with the project.
```

Comment:
@CecileRobertMichon Is there any precedent for distinguishing alpha vs. beta features in CAPZ?

Reply:
Switched to this Experimental vs Stable format